features-sdk/SecurityController.feature

```diff
@@ -36,3 +36,26 @@ Feature: Security Controller
     # Test of roles reset
     And The role "anonymous" should match the default one
     And The role "default" should match the default one
+
+  @deleteProfile
+  Scenario: Delete a profile
+    Given I create a role "test-role" with the following policies:
+    | document | { "actions":{ "create": true, "update": true }} |
+    And I create a profile "test-profile" with the following policies:
+    | policies | [ { "roleId": "test-role" } ] |
```

Comment:
Proposal for more simpler profile description:
```
And I create a profile "test-profile" with the following policies:
| test-role | { "index:" "example", "collections": ["one", "two"] } |
```
So we can deal with restrictions in the futur

Reply:
You proposal can't work
I think you meant
`| policies | [ { "roleId": "test-role", "restrictedTo": [ { "index": "example", "collections": ["one", "two"] } ] } ] |`

Reply:
It can work, you just have to code it differently :p
The syntax will be easier this way